### PR TITLE
Input Component

### PR DESCRIPTION
**Pull request by zbarnz**
This is the input component we will be using. Id like to add a few more things like animations and maybe colors (especially for declined inputs) but i wanted to get this out so it can be used. Here's a screenshot of the available variations: 

image

As of now you can choose between outlined and unoutlined, weather or not you want placeholders or labels, and size of the input box. **Dropdowns coming soon**

### CODE REVIEW

**Review by Dahvd — APPROVED** [review]
Sounds good.

File: www/components/Input.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+const Input = ({
+  label,
+  placeholder,
+  onChange,
+  value,
+  size,
+  required,
+  varient,
+}) => {
+  Input.propTypes = {
```
Comment (thudsonbu) [inline www/components/Input.js:14]: Do `propType` specs work inside of the component? The docs show something like this:
```js
import PropTypes from 'prop-types';

class Greeting extends React.Component {
  render() {
    return (
      <h1>Hello, {this.props.name}</h1>
    );
  }
}

Greeting.propTypes = {
  name: PropTypes.string
};
```

File: www/styles/LogReg.module.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
+.formContainer {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  position: relative;
```
Comment (thudsonbu) [inline www/styles/LogReg.module.css:5]: In the real app we should try not to use `position: relative` as it is very difficult to debug.

**Review by thudsonbu — APPROVED** [review]
Overall the `Input` component looks great thanks for doing that!

File: www/components/Input.js (hunk 1 above)
Comment (zbarnz) [reply www/components/Input.js:14]: They do, I think it probably makes more sense to have them outside though if that's what the docs say. Maybe some of the errors I was having were due to that

**Comment by Dahvd** [comment]
I just looked at this to get an idea of how it works at its awesome!

**Pull request merged**